gluten-ut/pom.xml

```diff
@@ -24,6 +24,11 @@
     <relativePath>../pom.xml</relativePath>
   </parent>
 
+  <modules>
+    <module>common</module>
+    <module>${spark.test.module}</module>
+  </modules>
```

Comment:
Does it work if just adding the below in shims/pom.xml,  similar to the above? 

```
  <modules>
    <module>common</module>
    <module>${spark.shim.module}</module>
  </modules>
```

Reply:
addessed and manually verified